src/table/src/table.rs

```diff
@@ -30,13 +38,29 @@ mod metrics;
 pub mod numbers;
 pub mod scan;
 
+lazy_static! {
+    //// The [`Expr`] to call UDF function `now()`.
+    static ref NOW_EXPR: Expr = {
+         let ctx = SessionContext::new();
+
+        let now_udf = ctx.udf("now").expect("now UDF not found");
+
+        Expr::ScalarFunction(ScalarFunction {
+            func: now_udf,
+            args: vec![],
+        })
```

Comment:
```suggestion
        let ctx = SessionContext::new();

        let now_udf = ctx.udf("now").expect("now UDF not found");

        Expr::ScalarFunction(ScalarFunction {
            func: now_udf,
            args: vec![],
        })
```

Reply:
It seems odd that `cargo fmt` does not work for this situation.